### PR TITLE
feat(external_api) Added tests for Av Moderation API

### PR DESCRIPTION
**Pull request by robertpin**

### CODE REVIEW

**Comment by damencho** [comment]
@robertpin Are you looking at the failures?

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by damencho** [comment]
Also running it locally I got:
```
[ERROR] Tests run: 403, Failures: 1, Errors: 0, Skipped: 394, Time elapsed: 85.188 s <<< FAILURE! - in TestSuite
[ERROR] testFunctionIsParticipantForceMuted(org.jitsi.meet.test.IFrameAPITest)  Time elapsed: 9.572 s  <<< FAILURE!
java.lang.AssertionError: Participant should be audio force muted expected [true] but found [false]
	at org.jitsi.meet.test.IFrameAPITest.testFunctionIsParticipantForceMuted(IFrameAPITest.java:878)

```

File: src/test/java/org/jitsi/meet/test/IFrameAPITest.java [hunk 1]
```diff
@@ -712,6 +712,239 @@ public void testFunctionIsAudioOrVideoMuted()
...
+        addIframeAPIListener(driver1, "moderationParticipantApproved");
+        addIframeAPIListener(driver1, "moderationParticipantRejected");
+
+        // enable audio and video moderation
+        TestUtils.executeScript(driver1,
+            "return window.jitsiAPI.executeCommand('toggleModeration', true);");
+        TestUtils.executeScript(driver1,
+            "return window.jitsiAPI.executeCommand('toggleModeration', true, 'video');");
+
+        TestUtils.waitMillis(500);
```
Comment (damencho) [inline src/test/java/org/jitsi/meet/test/IFrameAPITest.java:875]: Same

File: src/test/resources/files/iframeAPITest.html [hunk 2]
```diff
@@ -44,6 +44,15 @@
         window.jitsiAPI.addListener('videoConferenceJoined', function(event) {
             window.jitsiAPI.test.videoConferenceJoined = event;
         });
+        window.jitsiAPI.addListener('moderationStatusChanged', function(event) {
```
[the remark this answers is not in the document]
Comment (robertpin) [reply src/test/resources/files/iframeAPITest.html:47]: I added the to also test the events, but if they don't need to be tested I can remove them

**Pull request merged**